moment.parseFormat.js

```diff
@@ -55,6 +55,9 @@
   var regexYearShort = /\d{2}/;
 
   var regexFillingWords = /\b(at)\b/i;
+  
+  var regexUnixOffset = /\d{13}/;
```

Comment:
can we rename this to regexUnixMillisecondTimestamp? Also in the comment below (line 78)